### PR TITLE
Addon js

### PR DESCRIPTION
**Pull request by drmmr763**
Adds ability to include single javascript and css files from addon bundles.

### CODE REVIEW

**Comment by alanhartless** [comment]
The question I have about this, is does it prevent adding multiple of the same script/stylesheets to the head tag? Since the head remains static after browsing other pages due to ajax, will we end up with a bunch of the same asset files in the head and will that cause conflicts/issues.

I would suggest adding some checks into the loadScript and loadStylesheet functions that look something like

```
 if (typeof Mautic.headLoadedAssets == 'undefined') {
        Mautic.headLoadedAssets = {};
 } else if (typeof Mautic.headLoadedAssets[url] != 'undefined') {
        // URL has already been appended to head
       return;
}

// Note that asset has been appended
Mautic.headLoadedAssets[url] = 1;

// Rest of the function
```

**Comment by drmmr763** [comment]
Actually don't merge this. I see an issue in the code. I'll also fix the above item.

File: app/bundles/AssetBundle/Views/Asset/form.html.php [hunk 1]
```diff
@@ -13,6 +13,9 @@
     $view['translator']->trans('mautic.asset.asset.menu.new');
 $view['slots']->set("headerTitle", $header);
 $view['slots']->set('mauticContent', 'asset');
+
+echo $view['assets']->buildAddonScript('bundlename/assets/assets.js');
```
Comment (alanhartless) [inline app/bundles/AssetBundle/Views/Asset/form.html.php]: Just noticed that this should probably be removed ;-)

**Comment by alanhartless** [comment]
@drmmr763 Is one of the issues you mentioned include that it seems to be missing the helper function in AssetsHelper.php to load a stylesheet?

File: app/bundles/AssetBundle/Views/Asset/form.html.php (hunk 1 above)
Comment (drmmr763) [inline app/bundles/AssetBundle/Views/Asset/form.html.php]: Yep that's what I was hinting at in my comment above :p Don't know how that happened - I'll fix it.

**Comment by drmmr763** [comment]
See new PR at https://github.com/mautic/mautic/pull/293

**Pull request closed without merging**